api/src/main/java/io/kafbat/ui/sasl/azure/entra/AzureEntraOAuthBearerTokenImpl.java

```diff
@@ -0,0 +1,64 @@
+package io.kafbat.ui.sasl.azure.entra;
```

Comment:
`io/kafbat/ui/config/auth/azure` is a better place for these classes

Reply:
Updated